docs/guides/deploy-warp-route.mdx

```diff
@@ -35,6 +35,11 @@ hyperlane config create warp
 If your config looks correct, you can now skip to [Step 2: Deployment](#2-deployment).
 Or see below for details on how to define your config manually.
 
+If you want to use some custom chains in your warp route deployment use the --chains flag
```

Comment:
It might be good to reference the deploy hyperlane section here as that would be required to do this (somebody might think that just creating this config is sufficient)